server/webapp/WEB-INF/rails.new/webpack/models/shared/plugin_infos.js

```diff
@@ -142,11 +174,14 @@ PluginInfos.PluginInfo.SCM = function (data) {
 };
 
 PluginInfos.PluginInfo.SCM.fromJSON = (data = {}) => new PluginInfos.PluginInfo.SCM({
-  id:          data.id,
-  version:     data.version,
-  about:       About.fromJSON(data.about),
-  scmSettings: PluggableInstanceSettings.fromJSON(data.scm_settings),
-  imageUrl:    _.get(data, '_links.image.href'),
+  id:                 data.id,
+  status:             PluginInfos.PluginInfo.Status.fromJSON(data.status),
+  pluginFileLocation: data.plugin_file_location,
+  bundledPlugin:      data.bundled_plugin,
+  version:            data.version,
+  about:              About.fromJSON(data.about),
+  scmSettings:        PluggableInstanceSettings.fromJSON(data.scm_settings),
```

Comment:
There will be a plugin_settings for everything but task. I will send a PR to fix the representers. Any reason why the plugin index page should show scmSettings? There is no need to parse the API response for scm settings, package settings, repo settings, auth config settings, role settings, profile settings, etc., is there? We have separate pages for these. Unless the view for this page is going to be drastically different from the existing one, in which case, we should probably discuss as a team.